### ISSUE TITLE
Genome extension

### ISSUE THREAD

**Issue by joao1980** [nigyta/dfast_core#56]
Hi, 
Can the extensions .fa or .fasta be used?

Since they are the same type of file and I was not able to see that information from the help command.
Thanks!
Cheers.

**Comment by nigyta** [comment]
Yes, DFAST does not distinguish file types based on extensions, so it accepts any extension as long as the file is in FASTA format.
